### ISSUE TITLE
Update dot-binary-file web component to support maxFileLength attr

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#18019]
FROM: https://github.com/dotCMS/core/issues/17772

In a [previous issue](https://github.com/dotCMS/core/issues/17772) we make the backend to support `maxFileLength`.

In the `dot-binary-file` web component [we have the property](https://github.com/dotCMS/core-web/blob/master/projects/dotcms-field-elements/src/components/dot-binary-file/dot-binary-file.tsx#L86) but we don't do anything with it.

## Context
Now that we are passing ALL the fields variables as attributes [here](https://github.com/dotCMS/core-web/blob/master/projects/dotcms-field-elements/src/components/dot-form/utils/fields.tsx#L172) we are duplicating some, like [here](https://github.com/dotCMS/core-web/blob/master/projects/dotcms-field-elements/src/components/dot-form/utils/fields.tsx#L167) and [here](https://github.com/dotCMS/core-web/blob/master/projects/dotcms-field-elements/src/components/dot-form/utils/fields.tsx#L168)

## Acceptance Criteria

1. Make the `dot-binary-file` support `maxFileLength`, show message when file didn't meet the requirement.
2. Fix the duplicated attributes we're passing
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

**Comment by nollymar** [comment]
**PR Core Web:** https://github.com/dotCMS/core-web/pull/1917

**Comment by alfredo-dotcms** [comment]
QA:  dot-binary-file web component now supports `maxFileLength` attr, In the dot-binary-file web component we have the property `maxFileLength`

**Comment by alfredo-dotcms** [comment]
![image](https://user-images.githubusercontent.com/37185433/169906788-d9f7df21-c8c1-42c5-90c4-1b762ae4551a.png)
dot-binary-file

**Comment by nollymar** [comment]
**Internal QA:** Passed

Steps I used for testing: 
- I added a new field variable (`maxFileLength`) to the field Headshot of the Contact Form. The value is expressed in bytes.

<img width="599" alt="Screen Shot 2022-05-23 at 4 54 05 PM" src="https://user-images.githubusercontent.com/8741395/169912066-473d6c86-67fc-4fa0-8ce7-edb50b2f76cd.png">


- In the FE, I tried to upload a binary larger/shorter than the limit I had defined.


<img width="1539" alt="Screen Shot 2022-05-23 at 4 56 51 PM" src="https://user-images.githubusercontent.com/8741395/169912037-42751f5e-b88c-47f1-9016-95faeb1af9d2.png">
